### PR TITLE
Add get and set global options methods

### PR DESCRIPTION
**Pull request by gkatsev**
The final part of #1998

### CODE REVIEW

**Comment by gkatsev** [comment]
Oh, I need to update the sandbox.

File: src/js/video.js [hunk 1]
```diff
@@ -42,6 +43,11 @@ videojs.players = Player.players;
 
 videojs.extends = extendsFn;
 
+videojs.getGlobalOptions = () => options;
```
Comment (mmcc) [inline src/js/video.js]: If we're going to start using fat arrows, make sure to make a note in the ES6 page on the wiki.

**Comment by mmcc** [comment]
:+1:

File: src/js/video.js (hunk 1 above)
Comment (gkatsev) [inline src/js/video.js]: I'll add a note. I decided to use it here specifically because this function becomes so short as a fat arrow function.

**Comment by heff** [comment]
LGTM. Can you also add a note to the 5.0 changes wiki? This is actually a pretty significant change considering it's used in most example embeds.

https://github.com/videojs/video.js/wiki/5.0-Change-Details

**Comment by gkatsev** [comment]
I'll get to that soon.

**Pull request closed without merging**